.idea/misc.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="ExternalStorageConfigurationManager" enabled="true" />
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_13" default="true" project-jdk-name="13.0.2" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_13" default="true" project-jdk-name="13" project-jdk-type="JavaSDK">
```

Comment:
Please revert this change. To prevent this, I think you just need to name your local version of the JDK in IntelliJ "13.0.2" instead of "13" (which I thought you'd done previously, so weird this is happening).

Reply:
drr... thanks.